lib/caching/caching.rb

```diff
@@ -1,17 +1,19 @@
 class Caching
-  cattr_accessor :backend
+  class Caching
```

Comment:
I think it would make more sense to move this to `lib/caching.rb` instead of nesting the namespace.

Reply:
As discussed, I've left it as is for now and will investigate how we can refactor this later.